### PR TITLE
dark mode button: start to implement javascricpt

### PR DESCRIPTION
**Pull request by marvinRoellecke**
- add index.js
- add js folder
- add profile-page.css
- add dark mode button
  
body background will turn grey. but only on profile-page. If you switch to questions or bookmarks. the page will reload and background will be light.

### CODE REVIEW

File: css/global.css [hunk 1]
```diff
@@ -24,6 +24,10 @@ body {
   background-color: var(--color__background);
 }
 
+.body__dark {
+  background-color: var(--grey);
+}
+
```
Comment (onebarloop) [inline css/global.css]: According to BEM - shouldn't you name this class `.body--dark`?

**Review by onebarloop — APPROVED** [review]
Nice! If you manage to apply darkmode to all sites, tell me. I'm also working on it!

File: css/global.css (hunk 1 above)
Comment (marvinRoellecke) [reply css/global.css]: absolutely right!

**Comment by marvinRoellecke** [comment]
> Nice! If you manage to apply darkmode to all sites, tell me. I'm also working on it!

I really would like to :D

**Pull request merged**